pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+We encourage submitting predictions in all three formats and for all 24 molecules when possible. Incomplete submissions - such as for a subset of compounds - will also be accepted, but will not necessarily be evaluated together with the rest of the submissions. However, we would emphasize that omission of SEM estimates will not cause a submission to be regarded as incomplete.
+
+Names of the prediction files must begin with the name of the submission type (i.e., typeIII), and your name and must end with an integer indicating set of prediction. For example, if you want to submit one prediction file for type III prediction (macroscopic pKas), you might name it `typeIII-myname-1.csv`, where myname is arbitrary text of your choice. If you submit two prediction files of the same submission type, you might name them `typeIII-myname-1.txt` and `typeIII-myname-2.txt`.
+
+The file will be machine parsed, so correct formatting is essential.
+
+Lines beginning with a hash-tag (#) may be included as comments. These and blank lines will be ignored.
+
+The file must contain the following four components in the following order: your predictions, a name for your computational protocol, a list of the major software packages used, and a long-form methods description. Each of these components must begin with a line containing only the corresponding keyword: `Predictions:`, `Name:`, `Software:`, and `Method:`, as illustrated in the example files. More detailed instructions will be provided on the challenge submission site on the D3R website.  
+
+## Evaluation strategy for computational pKa predictions
+
+Macroscopic pKa predictions will be directly compared to experimental pKa measurements. Microscopic pKas and predicted microstates will be evaluated by comparison to each other and how well they recapitulate experimental macroscopic pKas, as well as pKa predictions of ACD which uses a data-trained pKa predictions method.
+
+## Anonymous versus public participation
+
+When you upload your submission, you will have the option of having it treated anonymously. Anonymous submission means that we may report on your predictions and methods, but not your identity. Public participations means we may also say who you are. Please note that, although we will work to protect the identity of anonymous participants, we cannot make any guarantees. You may use the D3R website to change your submission’s anonymous/public status until the challenge has closed. However, after the challenge has closed, you may not change its anonymous/public status.
```

Comment:
participations --> participants
also say who you are --> may also include identifying information
However, after the challenge has closed, you may not change its anonymous/public status. --> However, after the challenge has closed, the status can no longer be changed.

Reply:
Done.